erratatool.yml

```diff
@@ -85,36 +85,36 @@ boilerplates:
     description: |
       Red Hat OpenShift Container Platform is Red Hat's cloud computing Kubernetes application platform solution designed for on-premise or private cloud deployments.
 
-      This advisory contains the container images for Red Hat OpenShift Container Platform 4.8.z. See the following advisory for the RPM packages for this release:
+      This advisory contains the RPM packages / container images for Red Hat OpenShift Container Platform 4.8.z. See the following advisory for the container images / RPM packages for this release:
```

Comment:
Did we change this for 4.9+? I don't remember seeing it worded this way in the other PRs submitted. I know the style guide says not to use /.